### PR TITLE
feat(ProfileMaster): get profile master list added

### PR DESCRIPTION
**Pull request by This-Is-Prince**
Screenshot 2022-09-30 at 2 32 27 PM

### CODE REVIEW

File: service/profile_master_service.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	return &ProfileMasterService{
+		db: authDB,
+	}
+}
+
+func (s *ProfileMasterService) GetProfileMaster(ctx context.Context, req *pb.GetProfileMasterRequest) (*pb.ProfileMasterResponse, error) {
+	_, tenant := auth.GetUserIdAndTenant(ctx)
+
+	language := req.Language
+	if language == "" {
```
[the remark this answers is not in the document]
Comment (This-Is-Prince) [reply service/profile_master_service.go:30]: Sure

File: service/profile_master_service.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		return nil, status.Error(codes.InvalidArgument, "Language not specified")
+	}
+
+	profileMasterListChan, profileMasterListErrorChan := s.db.ProfileMaster(tenant).FindByLanguage(req.Language)
+	list := make([]*pb.ProfileMasterProto, 0)
+
+	select {
+	case profileMasterList := <-profileMasterListChan:
+		for _, profileMaster := range profileMasterList {
```
Comment (SaiNageswarS) [inline service/profile_master_service.go:39]: Use "github.com/jinzhu/copier" to copy from model to proto directly. No need to manually copy model to proto.

File: service/profile_master_service.go (hunk 1 above)
Comment (SaiNageswarS) [inline service/profile_master_service.go:30]: Let's default it to english.

**Review by SaiNageswarS — CHANGES_REQUESTED** [review]
Please close two minor comments and merge this PR.

File: service/profile_master_service.go (hunk 1 above)
Comment (This-Is-Prince) [reply service/profile_master_service.go:31]: Done

File: service/profile_master_service.go [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+	}
+
+	profileMasterListChan, profileMasterListErrorChan := s.db.ProfileMaster(tenant).FindByLanguage(language)
+	list := make([]*pb.ProfileMasterProto, 0)
+
+	select {
+	case profileMasterList := <-profileMasterListChan:
+		for _, profileMaster := range profileMasterList {
+			result := &pb.ProfileMasterProto{}
+			copier.Copy(result, profileMaster)
```
Comment (SaiNageswarS) [inline service/profile_master_service.go]: copier let's you copy entire list.. No for loop required even..

File: service/profile_master_service.go [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+	return &ProfileMasterService{
+		db: authDB,
+	}
+}
+
+func (s *ProfileMasterService) GetProfileMaster(ctx context.Context, req *pb.GetProfileMasterRequest) (*pb.ProfileMasterResponse, error) {
+	_, tenant := auth.GetUserIdAndTenant(ctx)
+
+	language := req.Language
+	if language == "" {
```
Comment (SaiNageswarS) [inline service/profile_master_service.go:31]: Pls check standard way of checking if a string is empty in golang.. may be do trim and check if len of string is 0...

**Review by SaiNageswarS — APPROVED** [review]

File: service/profile_master_service.go (hunk 4 above)
Comment (This-Is-Prince) [reply service/profile_master_service.go:31]: Ok

**Pull request merged**